.golangci.yml

```diff
@@ -75,5 +75,15 @@ linters:
 
 run:
   tests: true
+  build-tags:
+    - e2e
   skip-dirs:
     - internal/mocks
+
+issues:
+  # Excluding configuration per-path, per-linter, per-text and per-source
+  exclude-rules:
+    # Don't check for cyclomatic complexity on e2e file.
+    - path: e2e
+      linters:
+        - gocyclo
```

Comment:
[q] Could you explain me what this piece of code does?

Reply:
I am not sure what's the purpose of `issues`

Reply:
what it says at the top
>  # Don't check for cyclomatic complexity on e2e file.

Our e2e files are long and complex and were hitting this lint I don't think there's value to fix them given that they read fine as they are and is an artefact of having to create something and then delete it in the same test

Reply:
For clarity on `issues`, it's to ignore known issues on your code so you can add golint and not have to fix them all at once